src/components/Container.js

```diff
@@ -1,28 +1,55 @@
 import React from 'react';
+import Result from './Result';
+import calculate from '../logic/calculate';
 
-const Container = () => (
-  <div className="container">
-    <button type="button" className="col-4" id="result"> 0 </button>
-    <button type="button" className="bg-grey"> AC </button>
-    <button type="button" className="bg-grey"> +/- </button>
-    <button type="button" className="bg-grey"> % </button>
-    <button type="button" className="bg-orange"> + </button>
-    <button type="button" className="bg-grey"> 7 </button>
-    <button type="button" className="bg-grey"> 8 </button>
-    <button type="button" className="bg-grey"> 9 </button>
-    <button type="button" className="bg-orange"> x </button>
-    <button type="button" className="bg-grey"> 4 </button>
-    <button type="button" className="bg-grey"> 5 </button>
-    <button type="button" className="bg-grey"> 6 </button>
-    <button type="button" className="bg-orange"> - </button>
-    <button type="button" className="bg-grey"> 1 </button>
-    <button type="button" className="bg-grey"> 2 </button>
-    <button type="button" className="bg-grey"> 3 </button>
-    <button type="button" className="bg-orange"> + </button>
-    <button type="button" className="col-2 bg-grey"> 0 </button>
-    <button type="button" className="bg-grey"> . </button>
-    <button type="button" className="bg-orange"> = </button>
-  </div>
-);
+export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dataObj: { total: null, next: null, operation: null },
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
 
-export default Container;
+  // btnName: null,
+  handleClick = (event) => {
+    this.setState(
+      (prevState) => (
+        {
+          dataObj: calculate(prevState.dataObj, event.target.getAttribute('btn_name')),
+        }
+      ),
+    );
+  }
+
+  // this.setState((prevState) => calculate(prevState, buttonName));
```

Comment:
* I suggest you remove all uncommented code as it makes your codebase bulky and unprofessional